src/main/java/com/jalasoft/jfc/model/Convert.java

```diff
@@ -22,14 +22,8 @@
  * @author Oscar Lopez.
  * */
 public class Convert {
```

Comment:
you have to change the name of the class to ImageConvert

Reply:
Done